requirements.txt

```diff
@@ -5,3 +5,5 @@ gunicorn==19.10.0
 flask-cors==3.0.9
 requests>=2.20.0
 Mastodon.py==1.6.3
+lxml==5.1.0
+cssselect==1.2.0
```

Comment:
I don't see this being imported, is this a dependency of lxml? Is necessary for this to be declared here?

Reply:
don't worry, I just tested myself and I realized that this is necessary